README.rst

```diff
@@ -70,6 +70,23 @@ Documentation for Toga can be found on `Read The Docs`_.
 
 .. _Read The Docs: https://toga.readthedocs.io
 
+Financial support
+-----------------
+
+The BeeWare project would not be possible without the generous support of our financial
+members:
+
+.. image:: https://beeware.org/community/members/anaconda/anaconda-large.png
```

Comment:
I can't see the rendering until the other PR has been merged, but we should make sure this image isn't so large that it ends up unnecessarily dominating the whole page.

Reply:
Agreed. The image *should* be ~500px wide, which should fit without any problems.